Leetcode/Python/9_Palindrome_Number.py

```diff
@@ -0,0 +1,17 @@
+"""
+9. Palindrome Number
+Given an integer, return a boolean indicating whether it is a palindrome or not
+Difficulty :- Easy
+
+https://leetcode.com/problems/palindrome-number/
+Input: x = 121
+Output: True
+
+author:- aarifsk
+"""
+
+
+class Solution:
+    def isPalindrome(self, x: int) -> bool:
+        num = str(x)
+        return num == num[::-1]
```

Comment:
if **str** is not use, this could be considered
`class Solution:
    def isPalindrome(self, x: int) -> bool:
        size = len(str(x))
        left_divider = 10**(size-1) 
        y = x
        for i in range(size//2 + 1):
            left_digit = x // left_divider
            right_digit = y % 10
            
            if left_digit != right_digit:
                return False
            
            x = x % left_divider
            y = y // 10
            left_divider = left_divider // 10
        return True`